session/session.go

```diff
@@ -226,6 +227,13 @@ type session struct {
 	builtinFunctionUsage telemetry.BuiltinFunctionsUsage
 	// allowed when tikv disk full happened.
 	diskFullOpt kvrpcpb.DiskFullOpt
+
+	// stmtStats is used to count various indicators of each SQL in this session
+	// at each point in time. These data will be periodically taken away by the
+	// background goroutine. The background goroutine will continue to aggregate
+	// all the local data in each session, and finally report them to the remote
+	// regularly.
+	stmtStats *stmtstats.StatementStats
```

Comment:
This also exists in `s.sessionVars.StmtStats`, how about remove it here?

Reply:
Since the location where ExecCount is collected now has changed (moved from `session` to `executor`), we cannot access the internal fields of the `session struct`, so a public pointer is held here.

Reply:
Did you mean remove the private pointer inside the `session struct`? This seems to be ok, I will try.

Reply:
done